internal/cli/atlas/backup/snapshots/list.go

```diff
@@ -67,6 +68,8 @@ func ListBuilder() *cobra.Command {
 		Annotations: map[string]string{
 			"clusterNameDesc": "Name of the Atlas cluster that contains the snapshots you want to retrieve.",
 		},
+		Example: fmt.Sprintf(`  # Return a JSON-formatted list of snapshots for the cluster named myDemo 
+		  %s backups snapshots list myDemo --output json`, cli.ExampleAtlasEntryPoint()),
```

Comment:
Adjust spacing- should be two spaces indented (not from "Example" but from the start of the line)
```suggestion
  %s backups snapshots list myDemo --output json`, cli.ExampleAtlasEntryPoint()),
```